.github/workflows/deployment.yaml

```diff
@@ -186,6 +186,12 @@ jobs:
             fi
           done
 
+      - name: Deploy Synapse notebooks
+        run: |
+          if ! az synapse notebook show --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MPI"; then
+            az synapse notebook create --workspace-name phdi${TF_ENV}synapse${SHORT_CID} --name "Convert Parquet MPI" --file "scripts/Synapse/convertParquet.ipynb"
```

Comment:
Does this mean that the notebook filename should be renamed to `convertParquet`? I'm in favor of `convertParquetMPI` because we're also going to have a conversion script for the MCI.

Reply:
Good point! I'll make that change 👍